test/Confluent.Kafka.IntegrationTests/Tests/Consumer_Subscription.cs

```diff
@@ -45,21 +46,15 @@ public void Consumer_Subscription(string bootstrapServers)
 
             using (var consumer =
                 new ConsumerBuilder<byte[], byte[]>(consumerConfig)
-                    .SetRebalanceHandler((c, e) =>
+                    .SetPartitionsAssignedHandler((c, partitions) =>
```

Comment:
Would like to see a test where assign() is called from the handler, to see that the exception is properly thrown.

Reply:
I wrote these, but as noted the exception can't be caught and i'm deferring writing the exception wrapping / pass-through code until a later PR.